apps/console/src/public/deployment.config.json

```diff
@@ -1180,6 +1180,7 @@
             "applicationRequestPathAuthentication": false,
             "applicationSystemAppsSettings": false,
             "certificates": false,
+            "enablingBackupCodesForB2BUsers": true,
```

Comment:
```suggestion
            "backupCodesForSubOrganizations": true,
```

Reply:
Validate if any `legacyMode` configs are there in `default.json` as well.

Reply:
> Validate if any `legacyMode` configs are there in `default.json` as well.

legacyMode configs are not there in default.json.[1]

[1] https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

Reply:
My bad, i meant `infer.json`

https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.infer.json#L247

Reply:
Sent a PR

https://github.com/wso2/carbon-identity-framework/pull/5622